runbooks/001-Connecting-To-The-Database.md

```diff
@@ -0,0 +1,107 @@
+# 001 - Connecting to the database
+
+This is a runbook on how to connect and start querying the `hmpps-person-record` or `hmpps-person-match` database.
+
+Following the process from the cloud-platform [guide](https://user-guide.cloud-platform.service.justice.gov.uk/documentation/other-topics/rds-external-access.html#accessing-your-rds-database).
+
+## Table of contents
+
+1. [Prerequisites](#prerequisites)
+2. [Namespace](#namespace)
+3. [Establishing the connection](#establishing-the-connection)
+   1. [Creating the Port Forward Pod](#creating-the-port-forward-pod)
+   2. [Port forward to the database](#port-forward-to-the-database)
+4. [Connecting to the database](#connecting-to-the-database)
+
+## Prerequisites
+
+* Must have kubernetes access to the desired namespace
+* Must have `kubectl`, which can be installed [here](https://kubernetes.io/docs/tasks/tools/#kubectl)
+
+## Namespace
+
+For hmpps-person-record the namespaces are listed below:
+* `hmpps-person-record-dev`
+* `hmpps-person-record-preprod`
+* `hmpps-person-record-prod`
+
+## Establishing the connection
+
+To be able to connect to the database you need to port forward to the desired database. 
+To do this there must be a kubernetes port-forward pod available to establish the connection.
+
+To check if a port forward pod exists, which will be named either:
+* `person-match-port-forward-pod`
+* `person-record-port-forward-pod`
+
+Run the following command:
+
+```shell
+kubectl get pods -n <namespace>
+```
```

Comment:
`kubectl get pods -n hmpps-person-record-dev | grep port-forward`
might be easier to use?